cgi-bin/LJ/Web.pm

```diff
@@ -1997,6 +1997,15 @@ sub entry_form_entry_widget {
 }
 
 
+sub minsec_for_user {
+    my $user = LJ::load_user( shift );
```

Comment:
For performance and style, we usually use `$_[0]` instead of `shift`. It ends up being slightly faster since we don't need to manipulate the underlying array that was used to pass the parameters in.